BitwardenShared/UI/Platform/Application/Views/BitwardenUITextView.swift

```diff
@@ -0,0 +1,131 @@
+
+        /// Initializes a new `Coordinator` for the `BitwardenUITextView`.
+        ///
+        /// - Parameters:
+        ///    -  parent: The parent view that owns this coordinator.
+        ///    - calculatedHeight: The height of the text view.
+        ///
+        init(
+            _ parent: BitwardenUITextView,
+            calculatedHeight: Binding<CGFloat>
+        ) {
+            self.parent = parent
+            self.calculatedHeight = calculatedHeight
+        }
+
+        func textViewDidChange(_ uiView: UITextView) {
+            parent.text = uiView.text
+            parent.recalculateHeight(
+                view: uiView,
+                result: calculatedHeight
+            )
+        }
+    }
+
+    // MARK: Properties
+
+    /// The text entered into the text field.
+    @Binding var text: String
+
+    /// The calculated height of the `UITextView`. This value is dynamically updated based on the
+    /// content size, and it helps to adjust the height of the view in SwiftUI.
+    @Binding var calculatedHeight: CGFloat
+
+    /// Indicates whether the `UITextView` is editable. When set to `true`, the user can edit the
+    /// text. If `false`, the text view is read-only.
+    var isEditable: Bool = true
+
+    /// Creates and returns the coordinator for the `UITextView`.
+    ///
+    /// - Returns: A `Coordinator` instance to manage the `UITextView`'s events.
+    ///
+    func makeCoordinator() -> Coordinator {
+        Coordinator(self, calculatedHeight: $calculatedHeight)
+    }
+
+    // MARK: - UIViewRepresentable Methods
+
+    /// Creates and configures the `UITextView` for this view.
+    ///
+    /// - Parameter context: The context containing the coordinator for this view.
+    /// - Returns: A configured `UITextView` instance.
+    ///
+    func makeUIView(context: Context) -> UITextView {
+        let textView = UITextView()
+        textView.adjustsFontForContentSizeCategory = true
+        textView.autocapitalizationType = .sentences
+        textView.delegate = context.coordinator
+        textView.textColor = Asset.Colors.textPrimary.color
+        textView.isScrollEnabled = false
+        textView.isEditable = isEditable
+        textView.isUserInteractionEnabled = true
+        textView.isSelectable = true
+        textView.backgroundColor = .clear
+        textView.tintColor = Asset.Colors.tintPrimary.color
+        textView.textContainerInset = .zero
+
+        let verticalPadding: CGFloat = 4
+        textView.textContainerInset = UIEdgeInsets(top: verticalPadding, left: 0, bottom: verticalPadding, right: 0)
+        textView.textContainer.lineFragmentPadding = 0
+        textView.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+
+        let backupFont = UIFont.preferredFont(forTextStyle: .body)
+        let customFont = UIFont(name: FontFamily.DMSans.regular.name, size: 15)
+        textView.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: customFont ?? backupFont)
```

Comment:
🎨 I think you can do this and not deal with optionals:
```suggestion
        let customFont = FontFamily.DMSans.regular.font(size: 15)
        textView.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: customFont)
```